docs/wasmer-js/node-modules/examples/hello-world/hello-world.md

```diff
@@ -6,113 +6,166 @@ sidebar_label: Hello World
 
 [Full Example Project Source Code](https://github.com/wasmerio/docs.wasmer.io/tree/master/docs/wasmer-js/node-modules/examples/hello-world)
 
-In this example, we will run the WASI module [as-echo](https://github.com/torch2424/as-echo), using `@wasmer/wasi`. 
 
-The WASI module will echo "Hello World", and we will get the standard output from `/dev/stdout` using the `@wasmer/wasmfs` module. 
+In this example, we want to use the following call chain:
 
-This example will be run in the browser, bundled and served by [Parcel](https://parceljs.org/). 
+`JavaScript` --> `WebAssembly` --> `Some native "OS" function`
 
-However, `@wasmer/wasi` and `@wasmer/wasmfs` also work in Node, and the code examples from this guide can be used as a loose example, as long as the ES6 syntax is replaced with what would be the node equivalent.
```

Comment:
Yeah, its rather vague.  I'll remove this comment and re-explain this fundamental point.
My plan is to create equivalent examples for using wasmer-js from Node - but this will be a separate PR...